src/python_testing/TC_IDM_4_2.py

```diff
@@ -244,18 +319,23 @@ async def test_TC_IDM_4_2(self):
             targets=[Clusters.AccessControl.Structs.AccessControlTargetStruct(cluster=Clusters.BasicInformation.id)],
             subjects=[CR2_nodeid])
 
-        self.add_ace_to_dut_acl(CR1, CR2_limited_ace)
+        # Restore original DUT ACL
```

Comment:
add_ace_to_dut_acl writes the ACL, right? So why do you need to write the original first?

Reply:
Correct, will be removed.